woocommerce-myparcel.php

```diff
@@ -191,21 +204,21 @@ public function includes()
         /**
          * Instantiate classes when WooCommerce is activated
          */
-        public function load_classes()
+        public function load_classes(): void
         {
-            if ($this->is_woocommerce_activated() === false) {
-                add_action('admin_notices', [$this, 'need_woocommerce']);
+            if (! empty($this->checkInstalledWooCommerceVersion())) {
+                add_action('admin_notices', [$this, 'showRequiredWooCommerceNotification']);
 
                 return;
```

Comment:
Misschien kan je die return(s) weghalen, zou het juist wel fijn vinden als hij dan ook meteen laat zien dat mijn php versie niet geschikt is, voordat ik "investeer" in woocommerce installeren of bijwerken

Reply:
Als je de return(s) weghaalt en woocommerce is bijvoorbeeld niet geactiveerd, dan crasht heel je omgeving.

Reply:
Oh, dan is dat maar zo 😛